pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,332 @@
+                >Dua Lipa: Tiny Desk (Home) Concert</amp-fit-text
+              >
+            </a>
+          </div>
+          <img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/dua_wide-41767488aed1f75828f980b6b41339a0c65d4964.jpg?s=600"
+          >
+        </article>
+        <article class="article-slide">
+          <div class="article-description">
+            <a href="https://www.npr.org/sections/national/">
+              <h2>National</h2>
+            </a>
+            <a
+              href="https://www.npr.org/2020/12/03/942034617/time-names-its-kid-of-the-year-water-testing-scientist-gitanjali-rao"
+            >
+              <amp-fit-text height="70" min-font-size="20" max-font-size="20">
+                'Time' Names Its Kid Of The Year: Water-Testing Scientist
+                Gitanjali Rao
+              </amp-fit-text>
+            </a>
+          </div>
+          <img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/time201207_koty.coverfinal_wide-a7cff22dc41f55823d394817f2c1a0acfc105a66.jpg?s=600"
+            layout="intrinsic"
+          >
+        </article>
+        <article class="article-slide">
+          <div class="article-description">
+            <a href="https://www.npr.org/sections/book-reviews/">
+              <h2>Book Reviews</h2>
+            </a>
+            <a
+              href="https://www.npr.org/2020/12/04/941982629/maureen-corrigans-10-books-that-will-connect-you-in-a-socially-distant-year"
+            >
+              <amp-fit-text height="70" min-font-size="20" max-font-size="20">
+                Maureen Corrigan's 10 Books That Will Connect You In A Socially
+                Distant Year
+              </amp-fit-text>
+            </a>
+          </div>
+          <img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/corrigan2020_wide-26e0b93fd657db854e1ba2bdb7b18539ca72ecfa-s1100-c85.jpg"
+          >
+        </article>
+      </amp-base-carousel>
+    </div>
+  </body>
+</html>
+```
+
+See the [demo in action on Glitch](TODO).
+
+# Managing events
+
+Fully valid AMP installs event listeners on elements via the [`on` attribute](../learn/amp-actions-and-events.md) with the event and responding action as values. Bento AMP does not rely on this attribute. Instead, the AMP team will provide a JS API to manage and react to events. 
+
+During the developer preview, you can register component events and respond with defined actions using following syntax:
+
+`el.getApi().then(api => api.play())`
+
+The code below triggers [`amp-base-carousel`'s next action](../../../documentation/components/reference/amp-base-carousel.md#next) when executed. 
+
+```js
```

Comment:
Show HTML and JS together. Otherwise it's unnecessarily hard to reproduce. Also, `$('amp--base-carousel');` is not a browser API.